Dockerfile

```diff
@@ -14,6 +14,7 @@ ENV SUMMARY="Very Secure Ftp Daemon" \
     RELEASE=1 \
     ARCH=x86_64
 
+LABEL MAINTAINER "Dominika Hodovska" <dhodovsk@redhat.com>
```

Comment:
I don't want to nitpick badly, but can you use lower case characters in the label name? (meaning `maintainer`)